global/scripts/golang/test/run.sh

```diff
@@ -39,7 +39,13 @@ directories=$(echo $directories | sed 's/^ *//;s/ *$//')
 echo "Testing code in the following directories: $directories"
 
 # Run the tests
-go test -v -tags test,unit,integration \
+go test -v -tags test,unit \
+  -coverpkg="$(echo $directories | tr ' ' ',')" \
+  -covermode=count \
+  -coverprofile=coverage.txt \
+  $directories
+
+go test -p 1 -v -tags integration \
   -coverpkg="$(echo $directories | tr ' ' ',')" \
   -covermode=count \
   -coverprofile=coverage.txt \
```

Comment:
Is this file being replaced or appended? Because the testing result should be complete, with all the coverage, from unit testing and also integration testing. As far as I know.